CHANGES.rst

```diff
@@ -0,0 +1,17 @@
+Changelog
+*********
+
+Changelog entries for the development version are available at
+https://hyperspy.readthedocs.io/en/latest/user_guide/changes.html
+
+.. towncrier-draft-entries:: |release| [UNRELEASED]
+
+.. towncrier release notes start
+
+RosettaSciIO v0.0
```

Comment:
```suggestion
0.1.0.dev0 [UNRELEASED]
```
Assuming that the first tag we will make will be 0.1.0?

Reply:
Yes, I guess this would make sense. Though for this comment, we could also list it as `RosecettaSciIO initiation` or something similar to keep it separate from other changelog entries of version `0.1.0` that come after the split?